pkg/cmdutil/exec.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package cmdutil
+
+import (
+	"os"
+	"os/exec"
+)
+
+func CommandTTY(name string, arg ...string) *exec.Cmd {
```

Comment:
Why TTY?

Reply:
You are right, it only needs to bind stdout/err. 

I copy pasted this from devbox repo where stdin needs to be set for the CLI to output in a certain way. In this case it doesn't need it.